### PR TITLE
Add builders example

### PR DESCRIPTION
**Pull request by antoyo**

### CODE REVIEW

File: src/bin/builders.rs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+use std::env::args;
+
+fn build_ui(application: &gtk::Application) {
+    let window = gtk::ApplicationWindow::new(application);
+
+    window.set_title("First GTK+ Program");
+    window.set_border_width(10);
+    window.set_position(gtk::WindowPosition::Center);
+    window.set_default_size(350, 70);
```
Comment (sdroege) [inline src/bin/builders.rs]: Would be nice to also have a builder for the above things :)

Comment (antoyo) [reply src/bin/builders.rs]: There's no builder generated for `ApplicationWindow`.
To be able to do this, that would require the builder to generate methods for parent properties. I'll see if I can do that.

**Comment by EPashkin** [comment]
@antoyo This PR better be against "pending" branch

**Comment by sdroege** [comment]
@antoyo What's the status here? Would be good to have this updated against the pending branch and then merge it :)

File: src/bin/builders.rs (hunk 1 above)
Comment (sdroege) [reply src/bin/builders.rs]: ... which happened AFAIU now? :)

File: src/bin/builders.rs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    window.add(&button);
+
+    window.show_all();
+}
+
+fn main() {
+    let application = gtk::Application::new("com.github.gtk-rs.examples.basic",
+                                            Default::default())
+                                       .expect("Initialization failed...");
```
Comment (sdroege) [inline src/bin/builders.rs]: Also wants a run of `cargo fmt` :)

**Comment by sdroege** [comment]
@GuillaumeGomez Good for me now :)

**Comment by GuillaumeGomez** [comment]
Waiting for CIs in any case. :)

**Comment by GuillaumeGomez** [comment]
Thanks @antoyo!

**Pull request merged**